packages/badge/src/react/__specs__/__snapshots__/storyshots.spec.js.snap

```diff
@@ -2,22 +2,10 @@
 
 exports[`Storyshots appearance default 1`] = `
 <div
-  style={
-    Object {
-      "background": "#181818",
-      "backgroundSize": "cover",
-      "bottom": 0,
-      "left": 0,
-      "overflow": "scroll",
-      "position": "fixed",
-      "right": 0,
-      "top": 0,
-      "transition": "background 300ms",
-    }
-  }
+  data-css-claeju=""
 >
   <div
-    className="css-vdc22r"
+    className="css-index--badge-ydzcd8"
```

Comment:
Why has the selector changed? I haven't seen anything change in the packages/badge code that points me to a reason.

Separately, is this a good change?  I'm torn because it's more descriptive and makes for easy debugging on one hand, but it also increases total size for clients, who shouldn't need to debug our styles anyway. 🤷‍♂️